content/release-10-18-0/announcements-10-18-0.md

```diff
@@ -8,6 +8,12 @@ layout: bundle
 
 #### Planned
 
+##### Breaking change in the Inventory API -- change of the return code for delete of managed object
+
+As announced with [release 10.17](/release-10-17-0/announcements-10-17-0), as of release 10.19+, when delete of managed object and its dependencies doesn't finish right away, the platform will return 202 (Accepted) http status code instead of 204 (No content).
```

Comment:
```suggestion
As announced with [release 10.17](/release-10-17-0/announcements-10-17-0), as of release 10.19+, if the deletion of a managed object and its dependencies doesn´t finish immediately, the platform will return a 202 (Accepted) HTTP status code instead of 204 (No content).
```